phpunit.xml

```diff
@@ -1,4 +1,7 @@
 <phpunit bootstrap="vendor/autoload.php">
+  <php>
+    <ini name="xdebug.overload_var_dump" value="off"/>
```

Comment:
A large number of the tests failed since we seem to have xdebug enabled, which causes `var_dump`'s formatting to be different than what was expected. If you remove this you'll see in the failure diff it prints things like `'Standard input code:19:\n`. This is because we're running the tests on the command line rather than using files, which causes argv[0] to be "standard input code", and with xdebug enabled it prints the file name which is "standard input code". https://stackoverflow.com/questions/38375389/why-does-var-dump-show-filename-and-line-number